README.md

```diff
@@ -1,46 +1,429 @@
+    - [Using the Arc.js Library in your Code](#using-the-arcjs-library-in-your-code)
+      - [Importing Arc.js](#importing-arcjs)
+      - [Configuring Arc.js](#configuring-arcjs)
+      - [Working with DAOs and Arc Contracts](#working-with-daos-and-arc-contracts)
+  - [Working with DAOs](#working-with-daos)
+    - [Create a new DAO with all defaults](#create-a-new-dao-with-all-defaults)
+    - [Create a new DAO with founders](#create-a-new-dao-with-founders)
+    - [Create a new DAO with schemes](#create-a-new-dao-with-schemes)
+    - [Create a new DAO overriding the default voting machine](#create-a-new-dao-overriding-the-default-voting-machine)
+      - [Default root-level override for all schemes](#default-root-level-override-for-all-schemes)
+      - [Scheme-specific override](#scheme-specific-override)
+    - [Create a new DAO with a non-Universal Controller](#create-a-new-dao-with-a-non-universal-controller)
+    - [Create a new DAO with a non-default DaoCreator scheme](#create-a-new-dao-with-a-non-default-daocreator-scheme)
+    - [Get a previously-created DAO](#get-a-previously-created-dao)
+    - [Get the DAOstack Genesis DAO](#get-the-daostack-genesis-dao)
+  - [Using the Arc Contracts](#using-the-arc-contracts)
+    - [Categories of Arc Contracts](#categories-of-arc-contracts)
+      - [Schemes](#schemes)
+      - [Voting Machines](#voting-machines)
+      - [Global Constraints](#global-constraints)
+      - [Others](#others)
+    - [Obtain a DAO's Scheme](#obtain-a-daos-scheme)
+    - [Obtain a wrapper using the wrapper's factory class](#obtain-a-wrapper-using-the-wrappers-factory-class)
+      - [deployed](#deployed)
+      - [new](#new)
+      - [at](#at)
+    - [Obtain a wrapper from getDeployedContracts](#obtain-a-wrapper-from-getdeployedcontracts)
+    - [Obtain any Arc contract using Utils.requireContract](#obtain-any-arc-contract-using-utilsrequirecontract)
+  - [Working with Arc.js Scripts](#working-with-arcjs-scripts)
+  - [Deploying to Other Testnets](#deploying-to-other-testnets)
+  - [Running Against a Ganache Database](#running-against-a-ganache-database)
+  - [Contribute to Arc.js](#contribute-to-arcjs)
+  - [Security](#security)
+  - [License](#license)
 
-DAOstack is a widely open collaboration to build the basic framework for [Decentralized Autonomous Organizations](https://en.wikipedia.org/wiki/Decentralized_autonomous_organization) (DAO) through bootstrap, or dogfooding.
+## Getting Started
 
+### Installation
 
-# Contributing
+Ensure that [NodeJS](http://nodejs.org/), v9.4.0 or greater, is installed.
 
-Contributions and pull requests are very welcome. Join us on [Slack](daostack.slack.com).
+Install the npm package into your project:
 
-If you want to contribute to the code, check out  [CONTRIBUTING.md](CONTRIBUTING.md).
+```script
+npm install @daostack/arc.js --save
+```
+
+Now you can proceed with migrating contracts to a running testnet and configuring Arc.js in your application and environment.
```

Comment:
seems like an unnecessary sentence

Reply:
I prefer this.  It is good technical writing style to introduce where the reader is going next.